scripts/onboarding/templates/arc-k8s-extension-provision/existingClusterParam.json

```diff
@@ -0,0 +1,21 @@
+{

```

Comment:
Rename the directory from "arc-k8s-extension-provision-*" to "arc-k8s-extension-lcm-*" or "arc-k8s-extension-provisionedcluster-*"

Reply:
renamed